File: KoKoslan/src/kokoslan/java/ast/KoKoString.java
/** 
   Andrey Arguedas Espinoza
   Daniela Armas Sanchez
   Michael Chen Wang
   Kimberly Olivas Delgado
*/

package kokoslan.java.ast;
import java.util.*;
import java.io.*;
import kokoslan.java.eval.*;

public class KoKoString extends KoKoAtom<String>{
   
   public KoKoString(String value){
      super(value);
   }

   @Override
   public KoKoValue eval(KoKoContext ctx){
	   return new KoKoStringValue(getValue());  
   }
}

File: KoKoslan/src/kokoslan/kotlin/ast/KoKoString.kt
/**
Andrey Arguedas Espinoza
Daniela Armas Sanchez
Michael Chen Wang
Kimberly Olivas Delgado
 */

package kokoslan.kotlin.ast

import kokoslan.kotlin.eval.*

class KoKoString(value: String) : KoKoAtom<String>(value) {

    override fun eval(ctx: KoKoContext): KoKoValue {
        return KoKoStringValue(getValue())
    }

}